### DISCUSSION TITLE
how to access clientHeight?

### DISCUSSION THREAD

**Discussion by andrzejdzirba** [ueberdosis/tiptap#2885] [General Questions]
is it possible to get node's clientHeight for paragraphs ad other nodes? preferably in a way to be able to use it in addAttribute? For instance suchas below? 

` addAttributes() {
    return {
      height: {
        default: null,
        parseHTML: element => element.clientHeight,
        renderHTML: attributes => {
          return {
            'data-height': attributes.height,
            style: `height: ${attributes.height}`,
          }
        },
      },
    }
  },`

**Comment by dgtlntv** [comment]
Hey! Did you ever find out how to access clientHeight/Width? I am currently trying to achieve the same thing!
